som_facturacio_comer/giscedata_facturacio_contracte_lot.py

```diff
@@ -15,36 +15,36 @@ class GiscedataFacturacioContracteLot(osv.osv):
     _name = 'giscedata.facturacio.contracte_lot'
     _inherit = 'giscedata.facturacio.contracte_lot'
 
-    def _ff_total_incidencies(self, cr, uid, ids, name, args, context=None):
-        """ Retorna nombre d'incidències en contracte_lot """
-        res = dict.fromkeys(ids, 0)
-        for id in ids:
-            status = self.read(cr, uid, id, ['status'])['status']
-            if status:
-                r1 = re.findall(r"(\[[a-zA-Z]{1,2}[\d]{2,3}\])", status)
-                res[id] = len(set(r1))
-            else:
-                res[id] = 0
-        return res
+    _STORE_WHEN_INVOICE_ADDED = {
+        'giscedata.facturacio.contracte_lot': (
+            lambda self, cr, uid, ids, context=None: ids,
+            ['factures_ids'],
```

Comment:
```suggestion
            ['n_factures'],
```

El camp factures_ids no s'està setejant, no tinc clar perquè (una millora no aplicada, suposo).
Però el camp `n_factures` és un camp enter que s'edita cada vegada que s'afegeix o elimina una factura, crec que serviria